src/Table2Beta/SelectedRowsHeader.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from "react";
+
+import Cell from "./Cell";
+import "./SelectedRowsHeader.less";
+import { FlexBox } from "../../src";
+
+interface Props {
+  selectedRows: Set<any>;
+  contentType?: { singular: string; plural?: string };
+  actions: Array<ActionInput>;
+  allSelected: boolean;
+}
+
+export interface ActionInput {
```

Comment:
Can this be exported in Table.tsx instead? I just realized this interface can also be used for single actions as well!